### DISCUSSION TITLE
creating an issue from project beta generate display error

### DISCUSSION THREAD

**Discussion by npracht** [community/community#19806] [Projects and Issues]
Hello,

When i create an issue from the Project table beta, there is issue on display.
My issue is correctly displayed in my project:
<img width="648" alt="Capture d’écran 2022-07-01 à 10 31 12" src="https://user-images.githubusercontent.com/14075239/176856999-aa3191c5-54d8-4eae-ac06-2a9c21c4ab04.png">

But if i open issue, n the side bar project is not displayed:
<img width="438" alt="Capture d’écran 2022-07-01 à 10 31 43" src="https://user-images.githubusercontent.com/14075239/176857089-43632662-fc40-41f1-a233-41c35aaecfc4.png">

If i click dropdown to link project, it shows already in project.
<img width="436" alt="Capture d’écran 2022-07-01 à 10 31 50" src="https://user-images.githubusercontent.com/14075239/176857125-8eb400ae-c1af-464f-9c8b-31577927a793.png">

I tried refresh browser, etc. It happens on several issues created directly from the project (not created first and then added to project)

**Comment by huabin** [comment]
I tried on Chrome and Firefox just now, and it doesn't happen. 

Could you provide me with a little more information?

- OS Version
- Browser Version
- Reproduce steps

Thank you.
